spec/spec_helper.rb

```diff
@@ -1,24 +0,0 @@
-ENV["RAILS_ENV"] ||= "test"
```

Comment:
Thinking about this we should probably leave the overall spec folder and spec_helper in place, and just remove the test file itself, as Rspec was setup in a different PR

Reply:
I've added it back in 👍